### ISSUE TITLE
EC2 : Instance terminated on reboot

### ISSUE THREAD

**Issue by sjones4** [Corymbia/eucalyptus#117]
Instances are sometimes terminated during reboot.

**Comment by sjones4** [comment]
Example output from nc.log when the issue occurs:

```
2018-10-08 14:53:44  INFO | [i-0348d195] rebooting requested
2018-10-08 14:53:44  INFO | [i-0348d195] shutting down
2018-10-08 14:53:47  WARN | [i-0348d195] hypervisor reported previously running domain as Shutdown
2018-10-08 14:53:47  INFO | [i-0348d195] cleaning up state for instance
2018-10-08 14:53:48 ERROR | libvirt: Domain not found: no domain with matching name 'i-0348d195' (code=42)
2018-10-08 14:53:48  INFO | [i-0348d195] rebooting
2018-10-08 14:53:48 ERROR | libvirt: Unable to pre-create chardev file '/var/lib/eucalyptus/instances/work/AIDAAV6O6LEA4NLKWDXDW/i-0348d195/console.log': No such file or directory (code=38)
2018-10-08 14:53:48 ERROR | [i-0348d195] failed to restart instance
2018-10-08 14:53:53 ERROR | libvirt: Domain not found: no domain with matching name 'i-0348d195' (code=42)
2018-10-08 14:53:53 ERROR | [i-0348d195] failed to write instance XML to /var/lib/eucalyptus/instances/work/AIDAAV6O6LEA4NLKWDXDW/i-0348d195/instance.xml
2018-10-08 14:53:53  INFO | [i-0348d195] cleaning up state for instance
```

**Comment by sjones4** [comment]
Fixed by corymbia/eucalyptus#121

**Comment by sjones4** [comment]
There are a couple of `eucalyptus.conf` settings used by node controllers that now relate to instances being terminated on reboot:

```
NC_REBOOT_GRACE_PERIOD_SEC=65
NC_BOOTING_CLEANUP_THRESHOLD=65
```

If instances are taking over 65 seconds to reboot then these values can be increased to prevent reboot failures due to timeouts.
